src/amo/components/AddonMoreInfo/index.js

```diff
@@ -69,7 +70,8 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
       );
     }
 
-    let supportUrl = trimAndAddProtocolToUrl(addon.support_url);
+    let supportUrl =
+      addon.support_url && trimAndAddProtocolToUrl(addon.support_url.outgoing);
```

Comment:
If we're going to do this for all outgoing urls, maybe `trimAndAddProtocolToUrl` should support `OutgoingUrl` and do this check internally?

Reply:
It could?  If we only use `trimAndAddProtocolToUrl` with outgoing style urls.  (which it does seem to be the case currently).  Though, outgoing urls are always fully qualified anyway so maybe we can just delete `trimAndAddProtocolToUrl` instead?

Reply:
https://github.com/mozilla/addons-frontend/pull/9960/commits/ef7f1910ff2495bd28cd04ca181a80df187c9ab1